Modules/Challenges/test/Challenges.API.Tests.Integration/Models/RequestConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Challenges.API.Tests.Integration.Models;
+public class RequestConfiguration
+{
+    public AuthenticationParameters AuthenticationParameters { get; set; } = new AuthenticationParameters();
+    public bool IsAuthenticated { get; set; } = false;
```

Comment:
The property name is not ideal. A RequestConfiguration cannot be authenticated. Let's rather use something like "Authenticate". Or maybe an "AuthenticationMode" property of type "AuthenticationMode", which is an enum with the values "Authenticated" and "Anonymous".